includes/commerce_pos.transaction.inc

```diff
@@ -936,3 +937,80 @@ function _commerce_pos_override_confirm_validate($form, &$form_state) {
   // Add the product to the overridden items list.
   $form_state['out_of_stock_overridden_products'][$form_state['triggering_element']['#product_id']] = $form_state['triggering_element']['#product_id'];
 }
+
+/**
+ * Returns a modal form displaying each item in an order along with a checkbox
+ * for the user to add that item to the return transaction.
+ */
+function commerce_pos_return_select_item_popup(&$form, &$form_state, $order) {
+
+  // Load our generic modal form.
+  $form['commerce_pos_return_select_item_modal'] = commerce_pos_form_modal(array(
```

Comment:
I'm a bit curious why it works though... Maybe another form in the same request is including it. But what if this form is called alone? Would it still work? Would it hurt to include it twice? Might be better to be explicit. But I haven't tested it so not sure.